clusters/kepler.yaml

```diff
@@ -0,0 +1,30 @@
+# Refer: https://artifacthub.io/packages/helm/kepler/kepler?modal=install
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: kepler-repo
+  namespace: kepler  # TODO: fix some better place where to store
+spec:
+  interval: 2m
+  url: https://sustainable-computing-io.github.io/kepler-helm-chart
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: kepler
+  namespace: kepler
```

Comment:
```suggestion
  namespace: flux-system
```

Reply:
but the docs mention that it is installed under namespace kepler

Reply:
We can modify it refering to this https://sustainable-computing.io/installation/kepler-helm/
but do we want to?